src/abis/IWalletJumpstart.ts

```diff
@@ -0,0 +1,310 @@
+const jumpstart = {
+  abi: [
```

Comment:
this is exactly the same file contents as WalletJumpStart.json, except wrapped into an object with an `abi` property